recipes/rust-activation/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.40.0" %}
+
+package:
+  name: rust_{{ target_platform }}-suite
+  version: {{ version }}
+
+build:
+  number: 1
```

Comment:
(To save anyone else from checking the same thing.) This needs to start at `1` to match with what is already in `defaults`.